distribution/src/bin-regular/hz-cli.bat

```diff
@@ -6,7 +6,6 @@ if NOT DEFINED JAVA_HOME goto error
 set RUN_JAVA=%JAVA_HOME%\bin\java
 set HAZELCAST_HOME=%~dp0..
 set CLASSPATH="%HAZELCAST_HOME%\lib\*";%CLASSPATH%
-set JAVA_OPTS=%JAVA_OPTS% "-Dhazelcast.client.config=%HAZELCAST_HOME%\config\hazelcast-client.yaml"
```

Comment:
This would break the `hz-cli.bat submit ...`

Ideally we should do the same as here:

https://github.com/hazelcast/hazelcast/blob/9b9c719b123c9a583ef53fd48854a5e0746adf09/distribution/src/bin-filemode-755/hz-cli#L27-L38

Reply:
...checking...

Reply:
have pushed a commit that aligns Windows with Linux in a clean way